docs/components/backdrop.md

```diff
@@ -28,15 +33,15 @@ See [MDC-103 for Flutter](http://codelabs.developers.google.com/codelabs/mdc-103
 To learn more about animations in Flutter, see [flutter.io/animations](flutter.io/animations).
 
 ### From a designer’s point of view
-The backdrop is an opportunity to give users easy access to an important element of the app that is secondary to another element. The backdrop’s back layer is persistent yet unobtrusive. Users have immediate access to the actions and content relevant to the front layer.
+The backdrop gives users easy access to an important, secondary element of an app. Its back layer is persistent yet unobtrusive, giving users access to the actions and content relevant to the front layer.
 
 ### Why not a reusable code component?
-_The backdrop is complex._ The component uses multiple types of animations and may depend on app-specific models or information. The level of complexity makes it difficult to generalize for a variety of contexts and a poor candidate for a reusable code component.
+_The backdrop is complex._ The component uses multiple types of animations and may depend on app-specific models or information. This makes it difficult to generalize across a variety of contexts. It's not a good candidate for a reusable code component.
 
 _The appearance of the backdrop can change with context._
-* Some apps may only need a single front layer and back layer, containing the backdrop to a vertical back and forth interaction.
+* Some apps only need a single front layer and back layer, containing the backdrop to a vertical back and forth interaction.
```

Comment:
Yeah I didn't write this well. The front layer's content could change considerably depending on what a user does in the back layer to affect it.

Reply:
Which line does this comment refer to?

Reply:
found it